services/frontend/package.json

```diff
@@ -74,5 +59,24 @@
       "last 1 safari version",
       "IE 11"
     ]
+  },
+  "devDependencies": {
+    "babel-core": "^6.26.3",
+    "babel-loader": "^8.2.2",
+    "babel-plugin-import": "^1.13.3",
+    "babel-plugin-lodash": "^3.3.4",
+    "babel-preset-env": "^1.7.0",
+    "eslint": "^7.17.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-config-prettier": "^7.1.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-prettier": "^3.3.1",
+    "eslint-plugin-react": "^7.22.0",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "lodash-webpack-plugin": "^0.11.6",
+    "prettier": "^2.2.1",
+    "webpack": "^5.12.1",
```

Comment:
I don't see webpack as a dep before this change. why do we need it now?

Reply:
We needed it before but yarn does seem to display the errors. When I moved to npm it showed me the error. It's a requirement by the lodash bundle package that shrinks the size of lodash.